docs/src/pages/styles/typography/_typography.docs.scss

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
This page doesn't need to be added - it's identical to the core guidance https://github.com/CMSgov/design-system/blob/development/packages/design-system-docs/src/pages/styles/typography/_typography.docs.scss

Reply:
Thanks for the input! I think we do want to eventually add different guidance to this page because we have these custom link styles for help drawers and tooltips. I haven't added anything yet but will be in the future. If you want I can take out this page for now.

Reply:
Hey @babsmiles After some further exploring we do need to have the typography page locally as well as the links section - I've made this update, thanks!